manager/Makefile

```diff
@@ -32,6 +38,9 @@ endif
 container:
 	docker build -t $(PREFIX)/$(IMAGE):$(TAG) -f Dockerfile $(DIR)
 
+container-push:
```

Comment:
Why not do this the same way as [the expandybird makefile](https://github.com/kubernetes/deployment-manager/blob/master/expandybird/Makefile#L24)?